android/src/main/java/io/ably/flutter/plugin/generated/PlatformConstants.java

```diff
@@ -5,445 +5,444 @@
 
 package io.ably.flutter.plugin.generated;
```

Comment:
There's quite a big diff here, so I have formatted it using the java tool that this file was formatted with previously. This reduced the diff: 97a6dde2486b0b7ff1d467d57c3891c126b079fe